src/grab.cpp

```diff
@@ -74,14 +74,17 @@ bool game::grabbed_veh_move( const tripoint &dp )
 
     //vehicle movement: strength check. very strong humans can move about 2,000 kg in a wheelbarrow.
     int mc = 0;
-    int str_req = grabbed_vehicle->total_mass() / 100_kilogram; //strength required to move vehicle.
+    //strength required to move vehicle.
+    int str_points_req = grabbed_vehicle->total_mass() / 1_kilogram;
```

Comment:
Am I reading this right that moving a 25 kg (55 lb) vehicle requires 25 STR?

Reply:
It seems like you could change this to divide by `10_kilogram` and remove/reduce the other divisions. Whenever `str_points_req` is used, it's being divided by a multiple of ten.

Reply:
> Am I reading this right that moving a 25 kg (55 lb) vehicle requires 25 STR?

No, the math works out the same as before (100kg vehicle requires 10 str) but it's more granular. Rather than increasing by 10 every 100 kilograms, it's 1 every 10 kilos.

Reply:
Is there a better way to word the comment on line 77, then? I'm not sure how to parse it.

Reply:
This last commit might be a little more understandable

Reply:
Makes a lot more sense, thanks